spec/dummy/db/schema.rb

```diff
@@ -11,11 +11,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141202081811) do
+ActiveRecord::Schema.define(version: 20150112083118) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "atmosphere_action_logs", force: true do |t|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.